src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robotInfo = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${version}, wheels: ${wheels}`;
```

Comment:
```suggestion
      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
```
Though in this case `version` and `this.version` are the same, you should use here `this` to get the proper for the proper instance of object